packages/wallet/src/redux/protocols/funding/player-a/__tests__/scenarios.ts

```diff
@@ -22,7 +23,7 @@ import { EmbeddedProtocol } from '../../../../../communication';
 //
 // 1. Happy path: WaitForStrategyChoice
```

Comment:
Could we update the comment with the new scenario? (Or just remove it, I don't think it's really necessary)

Reply:
I think these comments are unnecessary, and are just a pain to keep up to date!